src/ITensorNetworks/peps.jl

```diff
@@ -0,0 +1,142 @@
+"""
+A finite size PEPS type.
+"""
+mutable struct PEPS
```

Comment:
I don't think this needs to be mutable.

Reply:
Also I was thinking about a more general definition for the `PEPS` type which would be a specialized version of a more general `ITensorNetwork` type with a specified `Square` lattice. I made an issue about it here: #12. This could be left for future work. The idea would be that it could provide generic functionality for creating a new network with the specified lattice, priming certain indices of that lattice, etc. which would work generically for any lattice or network connectivity, but would make use of the specified lattice to make those operations more efficient so you don't have to write a specialized version of `prime(::PEPS)`, `prime(::MPS)`, `prime(::PEPO)`, `prime(::TTN)`, etc.

Reply:
Mutable removed. We can fix this issue in the next PR.